sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/EventProcessorClientEventSource.cs

```diff
@@ -162,13 +170,17 @@ protected EventProcessorClientEventSource() : base(EventSourceName)
         /// <param name="eventHubName">The name of the Event Hub that the processor is associated with.</param>
         /// <param name="consumerGroup">The name of the consumer group that the processor is associated with.</param>
         /// <param name="errorMessage">The message for the exception that occurred.</param>
+        /// <param name="sequenceNumber">The sequence number associated with the checkpoint being written.</param>
+        /// <param name="offset">The offset associated with the checkpoint being written.</param>
         ///
-        [Event(25, Level = EventLevel.Error, Message = "An exception occurred while attempting to perform a checkpoint update for partition '{0}' by processor instance with identifier '{1}' for Event Hub: {2} and Consumer Group: {3}.  Error Message: '{4}'")]
+        [Event(25, Level = EventLevel.Error, Message = "An exception occurred while attempting to perform a checkpoint update for partition '{0}' by processor instance with identifier '{1}' for Event Hub: {2} and Consumer Group: {3} at Sequence Number: {5} Offset: {6}. Error Message: '{4}'")]
```

Comment:
Hmm yeah I remember the PR awhile ago where you and Josh talked about making sure the order of parameters didn't change, but I had thought it was okay for the message parameters to be in a different order. I'll have to go back and fix that here - https://github.com/Azure/azure-sdk-for-net/pull/39454

Reply:
You and I both thought that it was safe.   I did exactly this in a couple of places and then had to go and fix a bunch of stuff because it "worked" but broke the way that arguments + names are passed to callers.    If you're interested, my fixes were in #43210.

Reply:
Ok updated this and the ones from the previous PR